src/apis/platformvm/interfaces.ts

```diff
@@ -358,7 +358,8 @@ export interface ClaimAmountParams {
 }
 
 export interface UpgradePhasesReply {
-  SunrisePhase: number
```

Comment:
why remove sunrise phase?

Reply:
I was following this:
https://github.com/chain4travel/caminogo/blob/dev/vms/platformvm/camino_service.go#L1051

Is SunrisePhase still relevant?

Reply:
its active camino phase

Reply:
It's back, but made optional, with all the other phases